cpp/include/cudf/legacy/io_writers.hpp

```diff
@@ -75,5 +75,63 @@ class writer {
 };
 
 }  // namespace orc
+
+
+namespace parquet {
+
+/**---------------------------------------------------------------------------*
+ * @brief Supported compression algorithms for the Parquet writer
+ *---------------------------------------------------------------------------**/
+enum class compression_type { none, snappy };
+
+enum statistics_freq {
```

Comment:
Should this be an enum class instead of just an enum?

Reply:
It should. Will change.